js_modules/dagit/packages/core/src/runs/LogsToolbar.tsx

```diff
@@ -239,19 +246,27 @@ const DownloadLink = styled.a`
   }
 `;
 
+export type LogLevelCounts = Record<LogLevel, number>;
+
 const StructuredLogToolbar = ({
   filter,
+  counts,
   onSetFilter,
   steps,
 }: {
   filter: LogFilter;
+  counts: LogLevelCounts;
   onSetFilter: (filter: LogFilter) => void;
   steps: string[];
 }) => {
   const [copyIcon, setCopyIcon] = React.useState<IconName>('assignment');
   const logQueryString = logQueryToString(filter.logQuery);
   const [queryString, setQueryString] = React.useState<string>(() => logQueryString);
   const copyToClipboard = useCopyToClipboard();
+  const logCountFormat = new Intl.NumberFormat(navigator.language, {
```

Comment:
what is this?! 

are we trying to be consistent about using these?

Reply:
This is so that we can display stuff like `150K` or `2.1M` instead of showing the entire number. I think being consistent about this would be great -- do you know of other places in Dagit where we could do something like this?

Reply:
hey @hellendag  I think this call may actually be expensive, want to pull it out of the render method up into a file constant? (Just because this component renders a lot)

Reply:
A "Formatters.ts" file would be useful so we can be consistent

Reply:
Thanks for checking the perf on it -- I initially had it in a module-level const and put it back in here figuring it couldn't be that bad. 😆